### PR TITLE
Updated kafka in docker-compose

### PR DESCRIPTION
**Pull request by vgarvardt**
# What this PR changes:
Fixes https://github.com/hellofresh/kandalf/issues/39

### CODE REVIEW

**Comment by ElenaForester** [comment]
+1

File: docker-compose.yml [hunk 1]
```diff
@@ -1,47 +1,73 @@
...
-            - 6379
-        networks:
-            - kandalf
+  kafka:
+    image: wurstmeister/kafka:latest
+    restart: always
+    depends_on:
+      - zookeeper
+    ports:
+      - "9092"
```
Comment (aleksandrzhiliaev) [inline docker-compose.yml:31]: Is this file for local purpose? If yes then we probably need to bind local ports as we are doing that in hfkafka:
https://github.com/hellofresh/hfkafka/blob/master/docker-compose.yml#L36

Comment (vgarvardt) [reply docker-compose.yml]: ok

**Comment by aleksandrzhiliaev** [comment]
+1

**Pull request merged**